list

```diff
@@ -1290,7 +1287,6 @@ toi.in
 tokopedia.link
 tonyr.co
 topt.al
-toyota.us
```

Comment:
I think these four removals are not related and should be separated and moved to the `inactive` list :sweat_smile:

Reply:
Done, restored and updated to current state